src/egon/data/datasets/etrago_setup.py

```diff
@@ -238,12 +238,12 @@ class EgonPfHvStorage(Base):
     control = Column(Text)
     type = Column(Text)
     carrier = Column(Text)
-    p_nom = Column(Float(53))
+    p_nom = Column(Float(53), server_default="0.")
     p_nom_extendable = Column(Boolean)
-    p_nom_min = Column(Float(53))
-    p_nom_max = Column(Float(53))
-    p_min_pu = Column(Float(53))
-    p_max_pu = Column(Float(53))
+    p_nom_min = Column(Float(53), server_default="0.")
+    p_nom_max = Column(Float(53), server_default="inf")
+    p_min_pu = Column(Float(53), server_default="-1.")
+    p_max_pu = Column(Float(53), server_default="1.")
     p_set = Column(Float(53))
     q_set = Column(Float(53))
     sign = Column(Float(53))
```

Comment:
There are no default values for `sign` and `p_nom_extendable`

Reply:
I added these values in commit 2e59552c1440dc62a3390c099a9d965cdaa69e5d